src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Wasm/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@dansiegel do we want to include launchSettings.json for the WASM target?

Reply:
afaik there shouldn't be a single change for the launchSettings.json in this PR

Reply:
be sure to remove this. It is not related to the issue and is fixed in another PR